tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5287,7 +5299,7 @@ declare namespace ts {
         getName(): string;
         getDeclarations(): Declaration[] | undefined;
         getDocumentationComment(typeChecker: TypeChecker | undefined): SymbolDisplayPart[];
-        getJsDocTags(): JSDocTagInfo[];
+        getJsDocTags(checker: TypeChecker): JSDocTagInfo[];
```

Comment:
Also a breaking change, right?

Reply:
Actually, I think I'll make this optional and skip resolving links if it's not provided. I wasn't happy about requiring it, and [resolved] link info is not something everybody needs all that frequently.